src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-windows`
Test-Connection.Ping.can convert IPv4 addresses to IPv6 with -IPv6
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Test-Connection.Tests.ps1: line 124
124:             $result = Test-Connection $targetAddress -IPv6 -Count 1

```